app/controllers/awards_controller.rb

```diff
@@ -0,0 +1,32 @@
+class AwardsController < ApplicationController
+  before_action :require_admin, only: [:create]
+  before_action :set_award, only: [:show]
+
+  def index
+    @awards = Award.all
+    @award = Award.new
+  end
+
+  def show
+
+  end
+
+  def create
```

Comment:
ideally, we wanted this create method to be in an Admin namespace to test that you understood how to do this kind of thing, namespacing will be important in future modules.